frontend/fingertips-frontend/components/pages/chart/index.tsx

```diff
@@ -105,6 +105,8 @@ export function Chart({
                     searchState={searchState}
                     groupIndicatorData={groupData}
                     xAxisTitle="Year"
+                    yAxisTitle="Unit"
```

Comment:
I don't think the y-axis title should be the literal text `Unit`. According to AC2 on the ticket, I think it should be:
```suggestion
                    yAxisTitle={indicatorMetadata?.unitLabel ? `Value: ${indicatorMetadata?.unitLabel}` : ''}
```